core/modules/user/components/UserAccount.ts

```diff
@@ -150,25 +151,36 @@ export const UserAccount = {
           newPassword: this.password
         })
       }
+      updatedProfile = pick(updatedProfile, config.users.allowModification)
       this.exitSection(null, updatedProfile)
     },
     exitSection (event, updatedProfile) {
       this.$bus.$emit('myAccount-before-updateUser', updatedProfile)
-      if (!updatedProfile) {
-        this.currentUser = Object.assign({}, this.$store.state.user.current)
-        this.userCompany = this.getUserCompany()
-        this.changePassword = false
-        this.oldPassword = ''
-        this.password = ''
-        this.rPassword = ''
-        if (!this.userCompany.company) {
-          this.addCompany = false
+      userHooks.afterUserProfileUpdated(async (event) => {
```

Comment:
Check comment below at first please.
If it is type Task - let's append it there
if not let's remove parentheses

Reply:
it is type Task but we don't need to add it here. It's already added as generic type https://github.com/DivanteLtd/vue-storefront/pull/4929/files#diff-2cde0f6fb20851a4e391587c17b3dad1R22
parentheses removed :+1: